src/pages/Groups/Group2/styles/Screen1.styled.ts

```diff
@@ -16,8 +16,76 @@ export const Screen = styled.div`
   height: 100vh;
 `
 
-export const BottomNavBar = styled.div`
+export const SearchBar = styled.div`
+  position: flex;
+  top: 0;
+  max-width: 400px;
+  min-width: 300px;
+  width: 50vw;
+  height: 70px;
+  border: 1px solid green;
+  border-radius: 0 0 15px 15px;
+`
+
+export const SearchButton = styled.div`
   position: fixed;
+  top: 0;
+  right: 38%; /*Not too sure about this*/
```

Comment:
its okay. not a concern for now. there are other ways to do it.

u can make a container for search bar and search button. then use justify-content to push both elements to extreme end